src/applications/financial-status-report/utils/helpers.js

```diff
@@ -377,3 +378,12 @@ export const generateUniqueKey = (data, fields, index = null) => {
   }
   return keyParts.join('-');
 };
+
+export const firstLetterLowerCase = str => {
+  if (!str || str.length === 0) return '';
+  // Check if the string is in the ignoreFields array
+  if (ignoreFields.includes(str)) {
+    return str;
+  }
+  return str.charAt(0).toLowerCase() + str.slice(1);
+};
```

Comment:
This helper uses the ignore fields constants file to make sure we don't decapitalize certain words defined by the VA content styleguide